app/controllers/stats/data/backend_apis_controller.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class Stats::Data::BackendApisController < Stats::Data::BaseController
+  before_action :set_source
+
+  ##~ sapi = source2swagger.namespace("Analytics API")
+  ##
+  ##~ e = sapi.apis.add
+  ##~ e.path = "/stats/backend_apis/{backend_api_id}/usage.{format}"
```

Comment:
We shouldn't support XML for new API endpoints 👀 only JSON allowed 💃

Reply:
It reuses the implementation of `Stats::Data::BaseController` (same as `Stats::Data::ServicesController`), where XML is supported. No need to override only to remove for the API backend, IMO. Once we remove XML from the base, it’s going to be removed for all endpoints.

Reply:
I am aware of that and that's why I didn't say anything in the routes.rb or in any other part of the controller. But at least we shouldn't add the documentation for XML for the new endpoint as we are doing in this line. That is like inviting the customer to use it when we do not want that.
So here it should be just:
`##~ e.path = "/stats/backend_apis/{backend_api_id}/usage.json"`

Reply:
Done.